### PR TITLE
fix(curriculum): add instruction to include target=_blank attribute in step 64

### LINKED ISSUE #56079: Learn Basic CSS by Building a Cafe Menu - Step 64
### Describe the Issue

In step 64 you are asked to create an anchor element, the anchor element in step 65 has a `target` attribute, but there is no test for this in step 64.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-64

### Your code

```html
      <footer>
        <p><a href="https://www.freecodecamp.org">Visit our website</a></p>
      </footer>


```

### Expected behavior

The `target` element in step 65 should not come as a surprise. Step 64 should instruct to add it.

### Screenshots

_No response_

### System

NA

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by sulaiman-dev**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes [#56079](https://github.com/freeCodeCamp/freeCodeCamp/issues/56079)

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3ef6e0a81099d9a697b550.md [hunk 1]
```diff
@@ -43,6 +45,12 @@ Your new `a` element should link to `https://www.freecodecamp.org`. Remember tha
 assert(document.querySelector("footer > p > a")?.href === "https://www.freecodecamp.org/");
 ```
 
+Your new `a` element should have the `target` attribute set to `_blank`.
+
+```js
+assert(document.querySelector("footer > p > a")?.target === "_blank");
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3ef6e0a81099d9a697b550.md:51]: please use `assert.isEqual`

Comment (sulaiman-dev) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3ef6e0a81099d9a697b550.md]: updated the code to use `assert.isEqual` as requested. Please review the changes. Thanks!

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
The failing tests with `assert.isEqual is not a function` say I gave the wrong suggestion. Please find the right one: https://www.chaijs.com/api/assert/

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍 

Thank you for your PR!

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3ef6e0a81099d9a697b550.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ dashedName: step-64
 
 Inside the `footer`, add a `p` element. Then, nest an anchor (`a`) element in the `p` that links to `https://www.freecodecamp.org` and has the text `Visit our website`.
 
+Make sure that the `link` opens in a new tab by adding a `target` attribute with the value `_blank`.
```
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3ef6e0a81099d9a697b550.md:12]:
```suggestion
Make sure that the link opens in a new tab by adding a `target` attribute with the value `_blank`.
```
Link doesn't need backticks.

**Review by gikf — APPROVED** [review]

**Comment by ilenia-magoni** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Comment by sulaiman-dev** [comment]
> Thank you for your contribution to the page! 👍
> We are happy to accept these changes and look forward to future contributions. 🎉

Thank you! I'm glad I could contribute. Looking forward to more opportunities to help out! 😊

**Pull request merged**